### PR TITLE
Features : Fuse.Sensor

### PR DESCRIPTION
**Pull request by ichan-mb**
FuseJS Module for easy access to sensors on the device. Besides being able to read sensor data, this module also has features to monitor changes in states of the battery and network connectivity

This PR contains:
- [ ] Changelog
- [x] Documentation
- [ ] Tests

### CODE REVIEW

File: Source/Fuse.Sensor/Android/data/BatteryData.java [hunk 1]
```diff
@@ -0,0 +1,150 @@
...
+
+import android.os.BatteryManager;
+import java.util.Locale;
+
+import static android.os.BatteryManager.*;
+
+public class BatteryData {
+
+	protected final int level;
+    protected final int scale;
```
Comment (kusma) [inline Source/Fuse.Sensor/Android/data/BatteryData.java]: Looks like there's some mixed indentation here

File: Source/Fuse.Sensor/Android/sensors/AbstractSensor.java [hunk 2]
```diff
@@ -0,0 +1,91 @@
...
+import android.hardware.SensorEvent;
+import android.hardware.SensorEventListener;
+import android.hardware.SensorManager;
+import android.os.Build;
+import com.foreign.Uno.Action_Object;
+
+public abstract class AbstractSensor {
+
+	private final SensorManager mSensorManager;
+    private final Sensor mSensor;
```
Comment (kusma) [inline Source/Fuse.Sensor/Android/sensors/AbstractSensor.java]: Again, seems like some mixed indentation.

File: Source/Fuse.Sensor/iOS/IOSSensorProvider.uno [hunk 3]
```diff
@@ -0,0 +1,835 @@
...
+
+		void OnError(string error)
+		{
+			_OnDataError(error);
+		}
+	}
+
+		[Require("Xcode.Framework", "SystemConfiguration")]
+	[ForeignInclude(Language.ObjC, "iOS/sensors/FOPressure.h")]
+	extern(iOS) class IOSPressureProvider : ISensorTracker
```
Comment (kusma) [inline Source/Fuse.Sensor/iOS/IOSSensorProvider.uno]: Looks like the Require is indented wrong

File: Source/Fuse.Sensor/Android/data/BatteryData.java (hunk 1 above)
Comment (ichan-mb) [reply Source/Fuse.Sensor/Android/data/BatteryData.java]: oops! sorry about that, fix is on the way..

**Comment by AndrewEQ** [comment]
Awesomeness! 🎉 Now we just need some docs on how to use it lol

**Pull request merged**